modules/datagrid/72/launch/added/launch/infinispan-config.sh

```diff
@@ -321,8 +321,19 @@ function configure_cache() {
                <off-heap size=\"$evictionSize\" eviction=\"$evictionType\" strategy=\"$evictionStrategyType\" address-count=\"$addressCount\"/>"
     fi
 
-    local eviction="\
+    local memory="\
                     <memory> $CACHE_MEMORY_STORAGE_TYPE </memory>"
+  else
+    local storageType=""
+    if [ "$(find_env "${prefix}_CACHE_MEMORY_STORAGE_TYPE")" == "binary" ]; then
+      storageType="<binary/>"
+    elif [ "$(find_env "${prefix}_CACHE_MEMORY_STORAGE_TYPE")" == "off-heap" ]; then
+      storageType="<off-heap/>"
+    else
+      storageType="<object/>"
```

Comment:
I find it a bit confusing that the default here is `object` and in `infinispan_service_profiles.sh` it's `off-heap`, maybe the default here should be to not define any kind of storage?

Reply:
@danberindei If we're using the image without a service profile, e.g. with the old templates, then it's necessary for the image to have working default values.